### PR TITLE
Jhalackova ped 9518 z fcp disks

### PR DESCRIPTION
**Pull request by lvicoun**
### Description

Adding deployment on zFCP disks.

### Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### Is this (based on) existing content?

* old xml:id...
* old URL...

### CODE REVIEW

**Comment by lvicoun** [comment]
[Micro-deployment-raw-images-ibmz-zfcp_en.pdf](https://github.com/user-attachments/files/17325125/Micro-deployment-raw-images-ibmz-zfcp_en.pdf)

File: articles/deployment-raw-images-ibm-dasd.asm.xml [hunk 1]
```diff
@@ -229,13 +232,227 @@
...
+    </module>
+    <module resourceref="_legal"/>
+    <module resourceref="_gfdl">
+      <output renderas="appendix"/>
+    </module>
+  </structure>
+
+  <structure renderas="article" xml:id="deployment-raw-ibmz-zfcg" xml:lang="en">
+    <merge>
+      <title>Deploying &productnameshort; using Raw Disk Images on IBM Z</title>
```
Comment (dariavladykina) [inline articles/deployment-raw-images-ibm-dasd.asm.xml:295]:
```suggestion
      <title>Deploying &productnameshort; Using Raw Disk Images on &zseries;</title>
```

File: articles/deployment-raw-images-ibm-dasd.asm.xml [hunk 2]
```diff
@@ -80,7 +83,7 @@
   <!-- S T R U C T U R E -->
   <structure renderas="article" xml:id="deployment-raw-ibmz-dasd" xml:lang="en">
     <merge>
-      <title>Deploying &productnameshort; using Raw Disk Images on IBM Z DASD Disks</title>
+      <title>Deploying &productnameshort; using Raw Disk Images on IBM Z</title>
```
Comment (dariavladykina) [inline articles/deployment-raw-images-ibm-dasd.asm.xml:86]:
```suggestion
      <title>Deploying &productnameshort; Using Raw Disk Images on &zseries;</title>
```

File: concepts/deployment-ibm-procedure-overview.xml [hunk 3]
```diff
@@ -34,12 +34,17 @@
...
   </para>
   <orderedlist>
-    <listitem>
+    <listitem condition="dasd">
       <para>
         DASD disks activation&mdash;before you start the deployment, you need
         to ensure that all disks to be used are available and active. For
         details, refer to
-        <xref linkend="deployment-raw-ibmz-dasd-preparing-machine"/>.
+        <xref linkend="deployment-raw-ibmz-dasd-preparing-machine"/>        .
```
Comment (dariavladykina) [inline concepts/deployment-ibm-procedure-overview.xml:42]:
```suggestion
        <xref linkend="deployment-raw-ibmz-dasd-preparing-machine"/>.
```

File: concepts/deployment-ibm-procedure-overview.xml [hunk 4]
```diff
@@ -34,12 +34,17 @@
...
+      </para>
+    </listitem>
+    <listitem condition="zfcp">
+      <para>
+        Ensure that all disk are available and active. For details, refer to <xref linkend="deployment-raw-ibmz-zfcd"/>
```
Comment (dariavladykina) [inline concepts/deployment-ibm-procedure-overview.xml:47]:
```suggestion
        Ensure that all disks are available and active. For details, refer to <xref linkend="deployment-raw-ibmz-zfcd"/>.
```

File: tasks/deployment-preparing-machine-ibm.xml [hunk 5]
```diff
@@ -18,15 +18,15 @@
  xmlns:xlink="http://www.w3.org/1999/xlink"
  xmlns:trans="http://docbook.org/ns/transclusion">
   <info>
-    <title>Preparing the machine</title>
+    <title>Preparing the DASD discs</title>
```
Comment (dariavladykina) [inline tasks/deployment-preparing-machine-ibm.xml:21]:
```suggestion
    <title>Preparing the DASD disks</title>
```

File: tasks/deployment-preparing-machine-ibm-zfcp.xml [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+<!-- metadata is dealt with on the assembly level -->
+<topic xml:id="deployment-raw-ibmz-zfcd"
+ role="task" xml:lang="en"
+ xmlns="http://docbook.org/ns/docbook" version="5.2"
+ xmlns:its="http://www.w3.org/2005/11/its"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink"
+ xmlns:trans="http://docbook.org/ns/transclusion">
+  <info>
+    <title>Preparing the zFCP/SCSI discs</title>
```
Comment (dariavladykina) [inline tasks/deployment-preparing-machine-ibm-zfcp.xml:21]:
```suggestion
    <title>Preparing the zFCP/SCSI disks</title>
```

File: tasks/deployment-preparing-machine-ibm-zfcp.xml [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+    <meta name="maintainer" content="jsindelarova@suse.com" its:translate="no"/>
+    <abstract>
+      <!-- can be changed via merge in the assembly -->
+      <para>
+        The topic covers the preparation of zFCD disks for the deployment.
+      </para>
+    </abstract>
+  </info>
+  <para>
+    To prepare the machine to deploy on zFCP/SCSI discs, proceed as described below:
```
Comment (dariavladykina) [inline tasks/deployment-preparing-machine-ibm-zfcp.xml:34]:
```suggestion
    To prepare the machine to deploy on zFCP/SCSI disks, proceed as described below:
```

File: tasks/deployment-preparing-machine-ibm-zfcp.xml [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+TYPE       ID                                            ON  PERS  NAMES
+zfcp-host 0.0.fa00                                       yes yes
+zfcp-host 0.0.fc00                                       yes yes
+zfcp-lun  0.0.fa00:0x500507630b181216:0x4021400a00000000 yes no   sda sg0
+zfcp-lun  0.0.fc00:0x500507630b101216:0x4021400b00000000 yes no   sdb sg1
+    </screen>
+  </step>
+  <step>
+    <para>
+      In the output there should be a LUN device&mdash;a device of the type
```
Comment (dariavladykina) [inline tasks/deployment-preparing-machine-ibm-zfcp.xml:53]:
```suggestion
      In the output, there should be a LUN device&mdash;a device of the type
```

File: tasks/deployment-preparing-machine-ibm-zfcp.xml [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+  <step>
+    <para>
+      Note the information about LUN devices. The <literal>/dev/sda</literal> device is used
+      further for the deployment of &productname;.
+    </para>
+  </step>
+  <step>
+    <para>
+      Even though formatting of the disk, where &productname; will be deployed, is not necessary.
+      Check if the disk is empty and does not contain any partition. If there are any partitions,
```
Comment (dariavladykina) [inline tasks/deployment-preparing-machine-ibm-zfcp.xml:87]:
```suggestion
      Even though formatting the disk where &productname; will be deployed is unnecessary,
      check if the disk is empty and does not contain any partitions. If there are any partitions,
```

File: tasks/deployment-raw-procedure-ibmz.xml [hunk 10]
```diff
@@ -75,23 +75,51 @@
...
       <para>
         Start &productname; by booting the second DASD minidisk in an x3270
         terminal:
       </para>
 <screen>&prompt.root;ipl 101</screen>
     </step>
+    <step condition="zfcp">
+      <para>
+Open the x3270 terminal and define the device to load the system from. Use the LUN number of the
+<literal>/dev/sda</literal> device, but split it to 8-digit chunks. For example the command for LUN
```
Comment (dariavladykina) [inline tasks/deployment-raw-procedure-ibmz.xml:109]:
```suggestion
<literal>/dev/sda</literal> device, but split it to 8-digit chunks. For example, the command for LUN
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions here. Thanks!

**Pull request merged**